### PR TITLE
**Feature:** Add checkbox component

### PR DESCRIPTION
**Pull request by fabien0102**
# Summary
cb529cd3-71f0-4909-bf9b-99ed6df91397

Add a pretty checkbox component to show boolean values with style 😎 

# Related issue

#183 

# To be tested

Me
- [x] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

File: src/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  ::before {
+    content: "";
+    cursor: pointer;
+    position: absolute;
+    display: block;
+    top: 0;
+    left: 0;
+    width: 20px;
+    height: 20px;
+    border-radius: 2px;
```
Comment (TejasQ) [inline src/Checkbox/Checkbox.tsx]: `theme.borderRadius` is what you're after.

File: src/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  cursor: pointer;
+  height: 20px;
+  display: block;
+  margin-bottom: 10px;
+  padding-left: 30px;
+  line-height: 20px;
+  user-select: none;
+
+  font-size: 12px;
+  color: #545454;
```
Comment (TejasQ) [inline src/Checkbox/Checkbox.tsx]: `theme.color.text.default`

**Comment by fabien0102** [comment]
@peterszerzo I let you adjust all the styles you want, you are the master of this so be my guest, my branch is yours! :wink:

The original mockup:
image

**Comment by peterszerzo** [comment]
@fabien0102  I understand that there is a time pressure on this, so feel free to add todo flags for improvements I suggested that you will not have time for, and take a tune-up later.

**Comment by peterszerzo** [comment]
I apologize for the delay in responding to this, folks, I have been caught up with other work. @fabien0102 I would rather you finished up the branch - the fixes are one-liners and a todo comment anyway. That said, do let me know if you need any clarification.

**Pull request merged**